gradle/libs.versions.toml

```diff
@@ -21,6 +21,7 @@ jacoco = "0.8.10"
 hilt = "2.46.1"
 navigation_component = "2.7.6"
 
+
```

Comment:
[Request] Please remove this empty line.

Reply:
Okay!